readme.md

```diff
@@ -35,7 +35,7 @@ architecture combinations:
 
   * Windows (Windows XP or greater) - x86 and x86_64
   * Linux (most, if not all, distributions) - x86, x86_64, ppc64 and armv6l
-  * Mac OS X (10.04 or greater) - x86, x86_64 and ppc64
+  * Mac OS X (10.04 or greater) - x86, x86_64, ppc64 and Apple Silicon
```

Comment:
or 
```suggestion
  * Mac OS X (10.04 or greater) - x86, x86_64, ppc64 and Apple Silicon(based on the ARM64 architecture)
```

Reply:
we don't test ppc64 anymore afaik, maybe we should remove it from this list?